syntax = "proto3";

package user;

service userLogin{

rpc userCreate (User) returns (Response) {}
rpc userAuth (User) returns (Response) {}
rpc tokenAuth(Token) returns (Response) {}

}

message User{

string username = 1;
string password = 2;

}

message Response{

string response = 1;

}

message Token {

string token = 1;

}
